app/services/hyrax/workflow/abstract_notification.rb

```diff
@@ -1,5 +1,27 @@
 module Hyrax
   module Workflow
+    # Subclass AbstractNotification to create a notification.
```

Comment:
please specify that it's a notification that happens when a state transition occurs. You probably want to use the `@abstract` YARD tag too.